app/src/main/java/com/example/bookstore/view_model/CalendarOfWeekViewModel.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.bookstore.view_model
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.domain.i_repository.ICalendarRepository
+import com.example.domain.model.calendar.Calendar
+import com.example.domain.model.calendar.Job
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.catch
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onCompletion
+import kotlinx.coroutines.flow.onEach
+import kotlinx.coroutines.flow.onStart
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@HiltViewModel
+class CalendarOfWeekViewModel @Inject constructor(private val iCalendarRepository: ICalendarRepository) :
+    ViewModel() {
+    private val _calendarList = MutableLiveData<List<Calendar>>()
```

Comment:
Không thấy dùng cái này?

Reply:
.onEach {
                    _calendarList.value = it